apps/docs/src/resources/integrations/payments/paypal.md

```diff
@@ -157,6 +190,42 @@ In the example above we do a couple of things:
 
 This event is called when the user approves the payment process. It's the last step before the payment is captured.
 
+```ts
+  ...
+  // part of window.paypal.Buttons({}) params
+  onApprove: async (data: OnApproveData, actions: OnApproveActions) => {
+    // createOrder from useCheckout composable
+    orderCreated.value = await createOrder({
+      paypalOrderId: data.orderID,
+      // optional - can be acdc, applepay, googlepay or venmo
+      product: 'sbp',
```

Comment:
`spb`, not `sbp` ;) 
Also, please document which product for which payment method, since it may be important, because different paymentSources get filled.